assets/queries/ansible/azure/sqlserver_with_public_access/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "ansible_sqlserver_with_public_access",
+  "queryName": "SQLServer with Public Access",
+  "severity": "HIGH",
+  "category": "Network Security",
+  "descriptionText": "Check if both start and end 'ip_address' are equal to 0.0.0.0 (any IP).",
+  "descriptionUrl": "https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/sql_firewall_rule"
```

Comment:
change descriptionUrl to point to Ansible documentation

Reply:
Done